UI/src/assetTypeConfiguration.js

```diff
@@ -1318,26 +1386,24 @@
           var lifecycleValidations = [
             { values: [4, 5], validate: function (startDate, endDate) { return !_.isUndefined(startDate) && !_.isUndefined(endDate) && endDate >= startDate; }}
           ];
+          var fields = selectedAsset.get().propertyData;
 
-          var opposite_side_sign =  _.find( selectedAsset.get().propertyData, function(prop) { if (prop.publicId === "opposite_side_sign") return prop; });
+          var opposite_side_sign =  _.find( fields, function(prop) { if (prop.publicId === "opposite_side_sign") return prop; });
           if (_.isUndefined(opposite_side_sign) || _.isUndefined(opposite_side_sign.values[0]) || opposite_side_sign.values[0].propertyValue === "") {
             selectedAsset.setPropertyByPublicId('opposite_side_sign', '0');
           }
 
           var functionFn = _.find(validations, function(validation){ return _.includes(validation.types, parseInt(Property.getPropertyValue('Tyyppi', selectedAsset.get())));});
-          var suggestedBoxValue = !!parseInt(_.find(selectedAsset.get().propertyData, function(asset) { return asset.publicId === "suggest_box"; }).values[0].propertyValue);
+          var suggestedBoxValue = !!parseInt(_.find(fields, function(asset) { return asset.publicId === "suggest_box"; }).values[0].propertyValue);
```

Comment:
Shouldnt we check if values exists before do values[0] ?

Reply:
I don't think this step is necessary because the variable **values** is initiated by default with **values: [ {propertyValue: 0} ]** on the assetTypeConfiguration newAsset definition